orchestration/hca_orchestration/tests/e2e/conftest.py

```diff
@@ -114,7 +155,7 @@ def load_hca_run_config(dataset_name: str, dataset_info: DatasetInfo):
         "solids": {
             "pre_process_metadata": {
                 "config": {
-                    "input_prefix": "gs://broad-dsp-monster-hca-dev-test-storage/integration/ebi_micro/test_data"
+                    "input_prefix": "gs://broad-dsp-monster-hca-dev-test-storage/integration/schema_integration"
```

Comment:
The previous staging area was wonky and did not actually pass the hca import validation tool. This is a new tweaked one, with a fake project ID and faked project information.